packages/fxa-content-server/server/lib/beta-settings.js

```diff
@@ -0,0 +1,103 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { readFileSync } = require('fs');
+const { join } = require('path');
+const { createProxyMiddleware } = require('http-proxy-middleware');
+const config = require('./configuration');
+
+const env = config.get('env');
+const settingsIndexFile =
+  env !== 'development'
+    ? readFileSync(join('../../../', 'beta/settings', 'index.html'), {
```

Comment:
I got this location from where it looks like the build will exists [here](https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/webpack.config.js#L204). Is this location correct?

Reply:
try something like:
```
join(
  __dirname,
  '..',
  '..',
  config.get('static_directory'),
  'beta',
  'settings',
  'index.html'
)
```